### PR TITLE
Added Codeforces Dynamic Programming Problems

### PR DESCRIPTION
**Pull request by Shreyashm16**

### CODE REVIEW

File: DP/BearCompressing.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <queue>
+#include <deque>
+#include <stack>
+#include <set>
+#include <utility>
+#include <iterator>
+#include <sstream>
```
Comment (siddharth25pandey) [inline DP/BearCompressing.cpp:13]: Please use a Single header file **bits/stdc++**

File: DP/LittleGirlMaxXOR.cpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <queue>
+#include <deque>
+#include <stack>
+#include <set>
+#include <utility>
+#include <iterator>
+#include <sstream>
```
Comment (siddharth25pandey) [inline DP/LittleGirlMaxXOR.cpp:13]: Please use a Single header file **bits/stdc++**

File: DP/SimpleStrings.cpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <queue>
+#include <deque>
+#include <stack>
+#include <set>
+#include <utility>
+#include <iterator>
+#include <sstream>
```
Comment (siddharth25pandey) [inline DP/SimpleStrings.cpp:13]: Please use a Single header file **bits/stdc++**

File: DP/TwoSubstrings.cpp [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <queue>
+#include <deque>
+#include <stack>
+#include <set>
+#include <utility>
+#include <iterator>
+#include <sstream>
```
Comment (siddharth25pandey) [inline DP/TwoSubstrings.cpp:13]: Please use a Single header file **bits/stdc++**

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Shreyashm16 Do the suggested changes

**Comment by Shreyashm16** [comment]
I have done the modifications that you had asked for. Thanks for reviewing.

**Review by siddharth25pandey — APPROVED** [review]
@Shreyashm16 LGTM :)
Able to merge
Thanks!

**Pull request merged**